presets/seawater.yaml

```diff
@@ -0,0 +1,39 @@
+'@module': pyEQL.solution
+'@class': Solution
+'@version': 0.0.post1.dev699+g0764b1c
+solutes:
+  H2O(aq): 55.34455401423017 mol
+  Cl[-1]: 0.54425785619973 mol
+  Na[+1]: 0.4675827371496296 mol
+  Mg[+2]: 0.05266118052346798 mol
+  SO4[-2]: 0.02815187344845402 mol
+  Ca[+2]: 0.010251594148212317 mol
+  K[+1]: 0.010177468379526855 mol
+  HCO3[-1]: 0.0017126511769261989 mol
+  Br[-1]: 0.0008395244921424561 mol
+  B(OH)3(aq): 0.0003134669156396757 mol
+  CO3[-2]: 0.00023825904349479544 mol
+  B(OH)4(aq): 0.0001005389715937341 mol
+  Sr[+2]: 9.046483353663284e-05 mol
+  F[-1]: 6.822478260456777e-05 mol
+  CO2(aq): 9.515218476861173e-06 mol
+  OH[-1]: 8.207436858780224e-06 mol
+  H[+1]: 7.943282347242822e-09 mol
+volume: 1.0080615264452506 l
+temperature: 298.15 K
+pressure: 1 atm
+pH: 8.103487039827968
+pE: 8.5
+balance_charge:
+solvent: H2O(aq)
+engine: native
+database:
+  '@module': maggma.stores.mongolike
+  '@class': JSONStore
+  '@version': 0.57.4
+  paths:
+  - /workspaces/pyEQL/src/pyEQL/database/pyeql_db.json
+  read_only: true
```

Comment:
I had not thought of this before, but one small problem with `dumpfn` is that when the `database` is serialized, the data includes the local path, which will be machine-specific. We don't need to address right now as long as your tests pass, but we should open an issue report to think of a more robust way to address this. (This is mainly a note to myself)

A simple way would be to simply delete the `database` key from the yaml files, so that pyEQL uses the default one